GenumKit/Parsers/FontsFileParser.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  FontsFileParser.swift
+//  Pods
+//
+//  Created by Derek Ostrander on 3/7/16.
+//
+//
+
+import Foundation
+import AppKit.NSFont
+
+typealias Font = (name: String, variant: String)
+
+extension CFString {
+    var alphanumericString: String {
```

Comment:
I'm not sure why filtering the font families to only keep alphanumeric characters is necessary?

In Swift, a lot of stuff can be used as Swift identifies, even emojis or Japanese characters car be case identifiers.
Besides, that's the job of the template to filter and transform those strings (because people creating templates might want to use the full name of the font to generate a comment before the `case` for exemple), and that's what the Stencil filter `swiftIdentifier`will be used for in the Stencil template :wink: